### ISSUE TITLE
"You are now leaving 4chan. The link you clicked is https://www(...).com. Click [here] to continue."

### ISSUE THREAD

**Issue by Fasmea** [ccd0/4chan-x#598]
Option for security measures.

**Comment by ccd0** [comment]
Can't you just look at the URL before you click a link?

**Comment by Fasmea** [comment]
I almost accidentally clicked the link to ecpm. See the /qa/ thread I put in my updated post.

**Comment by ccd0** [comment]
I think that's just a normal ad server.
Probably the best solution if you're worried about user-submitted links to malicious sites is to turn off automatic linkification so you have to copy the URL.

**Comment by ccd0** [comment]
And to reduce exposure to malvertising (which does happen even through legitimate advertisers) you need to use an ad blocker, not a userscript.

**Comment by Fasmea** [comment]
You are right but I could still see this option in the future. I'm a very concerned person in general, so sorry if I'm overreacting.

>you need to use an ad blocker, not a userscript.

I use Ublock Origin and Scriptsafe on Chrome, I think I should be safe.
